src/main/java/com/google/opensesame/servlets/UserServlet.java

```diff
@@ -86,5 +134,5 @@ public PreparedQuery queryInDatabase(
     return datastore.prepare(new Query(entityName).setFilter(userFilter));
   }
 
-  // TODO: make error handling conform with Richie's error handling
+  // TODO: The same GitHub account can be used with multiple emails :(
```

Comment:
That's a good point. In the future you could implement a check to determine if any user already exists with a given GitHub ID.